src/Storage/Storage.Management/help/Get-AzDataLakeGen2Item.md

```diff
@@ -24,41 +24,41 @@ This cmdlet only works if Hierarchical Namespace is enabled for the Storage acco
 ## EXAMPLES
 
 ### Example 1: Get a directory from a Filesystem, and show the details
-```
-PS C:\> $dir1 = Get-AzDataLakeGen2Item -FileSystem "filesystem1" -Path "dir1/"
-PS C:\> $dir1
+```powershell
+$dir1 = Get-AzDataLakeGen2Item -FileSystem "filesystem1" -Path "dir1/"
+$dir1
 
```

Comment:
Why there are no "```output" tag for the cmdlet output?

Reply:
@v-yuzhichen please have a look

Reply:
Because there are multiple commands and outputs in one example, we put them together for the time being without separating the commands and outputs.

Reply:
We might should find a proper way to handle this kind of example.
The current one looks a little confuse, espicially when there are mutiple command and output, it might not be easy to identify which line is command, which line is output without "PS C:\> "

Reply:
Okay, we'll talk about how to change this example as soon as possible.

Reply:
This is a bug of the docs team.We'll get ready first. If you think it's okay, let's merge again.

Reply:
Is there any ETA to fix this in help doc?
Personally, If we can't fix this soon, should we still keep "PS C:> " in this kind of example, to differ command line and output?